aiogram/__init__.py

```diff
@@ -1,8 +1,9 @@
+from aiogram.dispatcher.flags import FlagGenerator
+
+from . import filters, handler
```

Comment:
Since this change is breaking, let's rename both folders as plural?
`filters, handler`  ->  `filters, handlers`